src/isNotRegExp.js

```diff
@@ -0,0 +1,23 @@
+import { complement } from 'ramda';
+
+import isRegExp from './isRegExp';
+
+/**
+ * Checks if value is complement of `RegExp` object.
+ *
+ * @func isNotRegExp
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.5.0|v2.5.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {Boolean}
+ * @see {@link RA.isRegExp|isRegExp}
+ * @example
+ *
+ * RA.isNotRegExp(1); //=> true
+ * RA.isNotRegExp(new RegExp()); //=> false
```

Comment:
Please add
```js
RA.isNotRegExp(/(?:)/); //=> false

Reply:
@Undistraction I ment to add them to @examples not tests. Sorry if I wasnt clear